src/Forms/Element/AbstractElement.php

```diff
@@ -0,0 +1,494 @@
+<?php
+/**
```

Comment:
Add empty line between 1st line and comment. Also add `declare(strict_types=1);` in all php files as default.